modules/dokkatoo-plugin/src/main/kotlin/internal/parseKotlinVersion.kt

```diff
@@ -0,0 +1,23 @@
+package dev.adamko.dokkatoo.internal
+
+/**
+ * Accepts a full version string that contains the major, minor
+ * and patch versions divided by dots, such as "1.7.10".
+ *
+ * Does NOT parse and store custom suffixes, so `1.8.20-RC2`
+ * or `1.8.20-dev-42` will be viewed as `1.8.20`.
+ */
+internal fun parseKotlinVersion(fullVersionString: String): KotlinVersion? {
```

Comment:
Please just use Kotlins official KotlinToolingVersion instead. 
Its not even experimental in current master

Reply:
Thanks for the tip @sellmair. Would using `KotlinToolingVersion` require a minimum Kotlin version of 1.9.10? I would prefer to keep compatibility with Kotlin 1.7.10, since that would mean Dokkatoo would be compatible Gradle v7 plugins.

Reply:
I misread the GitHub UI - `KotlinToolingVersion` is in Kotlin 1.7+ (and possibly more but I didn't check very hard)